app/components/card.tsx

```diff
@@ -9,14 +10,16 @@ interface CardProps {
     name: string;
     age: number;
     job: Job;
+    title: string;
 }
 
-const Card = ({ name, age, job }: CardProps) => {
+const Card = ({ name, age, job, title }: CardProps) => {
```

Comment:
**Suggestion:** The `title` prop should be added to the `Card.defaultProps` object.
```suggestion
const Card = ({ name, age, job, title }: CardProps) => {
    title = title || Card.defaultProps.title;
```